.travis.yml

```diff
@@ -14,7 +14,7 @@ before_install:
 - cp .travis.maven.settings.xml $HOME/.m2/settings.xml
 - sudo /etc/init.d/postgresql stop
 - docker run -e POSTGRES_USER=vertx -e POSTGRES_PASSWORD=password -e POSTGRES_DB=testdb -p 5432:5432 -d postgres
-- docker run -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -e MYSQL_USER=vertx -e MYSQL_PASSWORD=password -e MYSQL_DATABASE=testdb -p 3306:3306 -d mysql
+- docker run -e MYSQL_ROOT_PASSWORD=password -e MYSQL_USER=vertx -e MYSQL_PASSWORD=password -e MYSQL_DATABASE=testdb -p 3306:3306 -d mysql/mysql-server:5.6
```

Comment:
Is this something that needs to be changed in the README as well? We have `MYSQL_ALLOW_EMPTY_PASSWORD=yes` there